src/materials/shaders/pointcloud.vert

```diff
@@ -490,11 +503,32 @@ void main() {
 		}
 	#endif
 
+   	// ---------------------
+	// Y RANGE HIGHLIGHT
+	// ---------------------
+
+     #if defined dual_vertical_range_highlight
+        vec4 world = modelMatrix * vec4( position, 1.0 );
+
+        bool insideRange1 = verticalRange1[0] <= world.y && world.y <= verticalRange1[1];
+        bool insideRange2 = verticalRange2[0] <= world.y && world.y <= verticalRange2[1];
+
+        vec3 LuminanceWeights = vec3(0.299,0.587,0.114);
+        float luminance = dot(vColor,LuminanceWeights);
+        if (insideRange2) {
+            vColor = getSaturatedColor(vec3(1.0, 0.8196, 0.0), luminance*1.1, 0.5);
+        } else if (insideRange1) {
+            vColor = getSaturatedColor(vec3(0.2, 0.5843, 1.0), luminance, 0.5);
```

Comment:
would it be better to have these values as constants somewhere?

Reply:
Maybe, but because it's in shader code there's no obvious place to load constants from. And I didn't want to add new uniforms to the shader (because you then need additional lines for every uniform in `point-cloud-material.ts`), but maybe it'd be the way to go (then it could be controlled by the user of the library).